### PR TITLE
Update/add multimodal test resources

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Describe your changes
Add test_resources for multimodal tasks

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [x] Proposed changes are described in the CHANGELOG.md

- [x] CI Tests succeed and look good!

### CODE REVIEW

File: src/datasets/processors/subsample/script.py [hunk 1]
```diff
@@ -56,31 +71,76 @@
...
+
+# subsample obs
+adata_output = adata_input[obs_index].copy()
+if par["input_mod2"] is not None:
+    adata_output_mod2 = adata_mod2[obs_index].copy()
+
+# filter cells and genes
+if par["input_mod2"] is not None:
+    n_cells =  scprep.utils.toarray(adata_output.X.sum(axis=1)).flatten()
+    n_cells_mod2 =  scprep.utils.toarray(adata_output_mod2.X.sum(axis=1)).flatten()
```
Comment (rcannood) [inline src/datasets/processors/subsample/script.py:84]: It seems a bit redundant to install scprep just for this.

```suggestion
    n_cells =  adata_output.X.sum(axis=1).A.flatten()
    n_cells_mod2 =  adata_output_mod2.X.sum(axis=1).A.flatten()
```

File: src/datasets/processors/subsample/script.py [hunk 2]
```diff
@@ -56,31 +71,76 @@
...
+
+#  subsample vars
+adata_output = adata_output[:, var_ix].copy()
+if par["input_mod2"] is not None:
+    adata_output_mod2 = adata_output_mod2[:, var_ix_mod2].copy()
+
+# filter cells and genes
+if par["input_mod2"] is not None:
+    n_cells =  scprep.utils.toarray(adata_output.X.sum(axis=1)).flatten()
+    n_cells_mod2 =  scprep.utils.toarray(adata_output_mod2.X.sum(axis=1)).flatten()
```
Comment (rcannood) [inline src/datasets/processors/subsample/script.py:118]: Likewise

It seems a bit redundant to install scprep just for this.

```suggestion
    n_cells =  adata_output.X.sum(axis=1).A.flatten()
    n_cells_mod2 =  adata_output_mod2.X.sum(axis=1).A.flatten()
```

File: src/datasets/processors/subsample/config.vsh.yaml [hunk 3]
```diff
@@ -42,6 +42,10 @@ functionality:
 platforms:
   - type: docker
     image: ghcr.io/openproblems-bio/base_python:1.0.1
+    setup:
+      - type: python
+        pypi:
+          - scprep
```
Comment (rcannood) [inline src/datasets/processors/subsample/config.vsh.yaml:48]: See comments below

```suggestion
```

File: src/datasets/processors/subsample/script.py [hunk 4]
```diff
@@ -1,19 +1,22 @@
 import scanpy as sc
 import random
 import numpy as np
+import scprep
```
Comment (rcannood) [inline src/datasets/processors/subsample/script.py:4]: See comments below

```suggestion
```

**Review by rcannood — CHANGES_REQUESTED** [review]
Minor changes :)

**Comment by KaiWaldrant** [comment]
I agree, I used an alternative in my test script which didn't seem to work. didn't know about the `.A`

**Review by rcannood — APPROVED** [review]

**Pull request merged**